src/servers/Cargo.toml

```diff
@@ -68,7 +68,7 @@ openmetrics-parser = "0.4"
 opensrv-mysql = "0.7.0"
 opentelemetry-proto.workspace = true
 parking_lot = "0.12"
-pgwire = "0.20"
+pgwire = "0.20.2"
```

Comment:
Cargo will ignore the patch version you specified and automatically update to 0.20.2 in Cargo.lock when you run `cargo update pgwire`. Please see https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio

Reply:
Here I need to specify this changeset only works for pgwire >= 0.20.2 because the patch version ships key fixes for this feature. This will make cargo.lock reproducible.

Reply:
> Here I need to specify this changeset only works for pgwire >= 0.20.2 because the patch version ships key fixes for this feature. This will make cargo.lock reproducible.

But this change does not actually pin the patch version. To do this you need:
```
pgwire = "=0.20.2"
```

Reply:
No, this will pin pgwire to 0.20.2 forever, which can be a resolver issue when there is a version conflict. All we need is 0.20.2 or later.